layersvt/device_simulation.cpp

```diff
@@ -103,26 +144,57 @@ std::string GetEnvarValue(const char *name) {
     return value;
 }
 
+#if defined(__ANDROID__)
+void AndroidPrintf(const char *level, const char *fmt, va_list args) {
```

Comment:
Yeah, the print routines are a bit messy.  Maybe just have wholly different Android functions for `DebugPrintf` an `ErrorPrintf`.  We can refactor them into `VulkanTools/common` and migrate the other layers to use them.  I see screenshot has picked up at least one `fprintf` getting dropped on Android.  I don't think we need to hold up this PR for that though.  We may even want that to happen upstream.